eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -19,344 +19,336 @@ parameters:
     default: true
   - name: CoverageReportPath
     type: string
-    default: 'sdk/*/*/*cov_xml.xml'
+    default: "sdk/*/*/*cov_xml.xml"
 
 jobs:
-- job: Validate
-  condition: and(succeededOrFailed(), ne(variables['Skip.Test'], 'true'))
-  strategy:
-    matrix:
-      Linux_x64_gcc8:
-        Pool: azsdk-pool-mms-ubuntu-1804-general
-        OSVmImage: MMSUbuntu18.04
-        VcpkgInstall: 'curl[ssl] libxml2 openssl'
-        VCPKG_DEFAULT_TRIPLET: 'x64-linux'
-        CC: '/usr/bin/gcc-8'
-        CXX: '/usr/bin/g++-8'
-        BuildArgs: '-j 10'
```

Comment:
If you ignore the whitespace changes in this PR, this is the substantive change here, which is removing the linux gcc8 leg from the matrix.

Reply:
@vhvb1989 can you please file an issue to track why the gcc8 leg is failing. I am assuming removing it from CI is a temporary measure to unblock the current PRs.

Do we plan to investigate and bring this leg back?